README.md

```diff
@@ -26,11 +27,19 @@ yarn
 yarn build
 ```
 
+#### Configure
+
+You need WalletConnect Cloud `projectId` to run Hyperlane Warp Route UI successfully. 
+Sign up to [WalletConnect Cloud](https://cloud.walletconnect.com/), create 
+new project with AppKit and Next.js and copy `projectId` from there.
+
 ### Run
 
+You need to set up environment variable `NEXT_PUBLIC_WALLET_CONNECT_ID` to `projectId` you have got from WalletConnect
+Cloud.
 ```sh
 # Start the Next dev server
-yarn dev
+export NEXT_PUBLIC_WALLET_CONNECT_ID=<projectId>; yarn dev
```

Comment:
Thanks for documenting this! Our original intention was for users to create a `.env.local` file next to the `.env.example` file in the root of the repo, though this `export` would also work. Would you mind mentioning the `.env.local` option?

Reply:
Yes, I can.